### PR TITLE
add back url parameter

### PR DESCRIPTION
**Pull request by rndmh3ro**
cc @felixfontein

### CODE REVIEW

**Comment by felixfontein** [comment]
You also need to update the documentation so it is marked as required. You can for example do that by adding:
```
    url:
        required: true
```
to the options documentation. That way, the doc fragment's values are used except for the `required` value.

**Comment by rndmh3ro** [comment]
> 
> 
> You also need to update the documentation so it is marked as required. You can for example do that by adding:
> 
> ```
>     url:
>         required: true
> ```
> 
> to the options documentation. That way, the doc fragment's values are used except for the `required` value.

That's an even better way than I just did. I'll change it again.

**Comment by rndmh3ro** [comment]
Should be good now.

File: plugins/doc_fragments/common_options.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class ModuleDocFragment(object):
+    # Documentation for common options that are always the same
+    DOCUMENTATION = r"""
+    options:
+      url:
+        description:
+          - HTTP or HTTPS URL in the form (http|https://[user[:pass]]@host.domain[:port]/path
+        required: true
+        type: strA
```
Comment (felixfontein) [inline plugins/doc_fragments/common_options.py]:
```suggestion
        type: str
```

**Review by felixfontein — APPROVED** [review]

**Pull request merged**